_sass/core/global.scss

```diff
@@ -19,6 +19,16 @@ body > div.content {
   padding: 0 13rem;
   @media screen and (max-width: 1170px) { padding: 0 6rem; }
   @media screen and (max-width: 480px) { padding: 0 2rem; }
+  @media screen and (max-width: 768px) {
+    padding: 0 1.2rem;
+    .width-10-12 {
+      grid-column: span 12;
+    }
```

Comment:
I wonder if we are making good use of the grid system because this sort of things should be automatic.

Reply:
It looks fine on desktop using only 10 columns of the 12, but on mobile you want to use all of them

Reply:
So the right fix would be to add `width-12-12-m` on the div in addition to `width-10-12`. That will solve the issue. Let's avoid fiddling with the grid system :).

BTW, we should also probably adjust `.navigation-wrapper` for mobile and tablet. Would you be interested in having a look?

Reply:
right there seems to be some big padding below it, 63e01117ea8aaa2e2f307462d5051f760142c76e should fix that